extractor/restriction_parser.cpp

```diff
@@ -152,6 +153,21 @@ RestrictionParser::TryParse(const osmium::Relation &relation) const
             {
                 is_only_restriction = true;
             }
+            if (strlen(iter->key()) > 11)

```

Comment:
For readability sake, maybe replace `11` with `const std::string("restriction").size()`.  Throwing the `const` in there should avoid any runtime overhead, and it shows the intent a bit more clearly.

Alternatively, throw a comment in there that describes why 11, like `// If key is > 11, then it will be a conditional restriction`